apps/web/app/constants.ts

```diff
@@ -116,3 +116,9 @@ export const APPLICATION_LANGUAGES_CODE = [
 	'ru',
 	'es'
 ];
+
+export enum KanbanView {
```

Comment:
Maybe let's rename this to "IssuesView"?

Reply:
sure I will rename that

Reply:
Btw, I think values should be:
CARDS, TABLE, and BLOCKS, please adjust @YehualashetGit  :)

Reply:
sure